.travis.yml

```diff
@@ -42,6 +43,7 @@ addons:
 
 env:
   global:
+    - DATABASE_URL=postgres://postgres:@localhost/cargo_registry_test
     - TEST_DATABASE_URL=postgres://postgres:@localhost/cargo_registry_test
```

Comment:
I was also using `DATABASE_URL` for compilation before, but since we don't need that now I can just specify `--database-url` instead